htdocs/fichinter/class/api_interventions.class.php

```diff
@@ -93,6 +94,7 @@ public function get($id)
 		}
 
 		$this->fichinter->fetchObjectLinked();
+		$this->fichinter->contacts_ids = $this->fichinter->liste_contact(-1, $contact_type, 1);
```

Comment:
Becausecthe contacts needs a subquery it should be done onlybif expected. So i suggest to have this parameter set to empty string '' by default. The load should be done only if param is provided.

Reply:
Done